ui/main/src/app/views/customCardList/ResultTable.ts

```diff
@@ -214,6 +220,40 @@ export class ResultTable {
         return this.typesOfState.includes(type);
     }
 
+    private isCardReadAndAck(card: Card): boolean {
+        if (!this.readAndAck || this.readAndAck.length === 0) return true;
+        const filterAcknowledged = this.readAndAck.includes(ReadAndAckEnum.ACKNOWLEDGED);
+        const filterNotAcknowledged = this.readAndAck.includes(ReadAndAckEnum.NOT_ACKNOWLEDGED);
+        const filterRead = this.readAndAck.includes(ReadAndAckEnum.READ);
+        const filterNotRead = this.readAndAck.includes(ReadAndAckEnum.NOT_READ);
```

Comment:
For performance reason , process this once using class private variable when 'setReadAndAckFilter'